content/14-passing-arguments-to-another-function-in-javascript/post.md

```diff
@@ -0,0 +1,27 @@
+Title: Passing arguments to another function in JavaScript
+
+-----
+
+Date: 1428527874
+
+-----
+
+Contributors: rasshofer
+
+-----
+
+Text:
+
+If you want to pass arguments to another function in JavaScript, you can use `.apply()` to have the same access to `arguments`.
+
+```js
+function a() {
+    b.apply(this, arguments);
+}
+
+function b() {
+    console.log(arguments);
+}
+
+a('Lorem', 'ipsum');​
+```
```

Comment:
I think it would make more sense to publish a separate post about the `arguments` object.